src/Chunk.h

```diff
@@ -392,18 +392,37 @@ class cChunk :
 	{
 		return m_ChunkData.GetMeta(a_RelX, a_RelY, a_RelZ);
 	}
-	inline void       SetMeta(int a_RelX, int a_RelY, int a_RelZ, NIBBLETYPE a_Meta)
+	inline void  SetMeta(int a_RelX, int a_RelY, int a_RelZ, NIBBLETYPE a_Meta)
```

Comment:
I've been wondering about this too. My reasoning was that `SetMeta` is a heavily used function, and for the sake of speed, one should skip the two extra ifs of `SetMetaQuiet` when possible. I have no idea if this argument holds in practice.